### PR TITLE
[doc] change config.sh to gen-config-yaml.sh #67

### LINKED ISSUE #67: [doc] change `config.sh` to `gen-config-yaml.sh`
https://github.com/apache/incubator-apisix-docker/pull/66#discussion_r459812151

### PR DESCRIPTION
**Pull request by gxthrj**
change config.sh to gen-config-yaml.sh

Fix #67

### CODE REVIEW

File: example/gen-config-yaml.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+export etcd_url='http://172.18.5.10:2379'
```
Comment (membphis) [inline example/gen-config-yaml.sh:3]: Shouldn't there be a fixed IP?

**Comment by membphis** [comment]
@gxthrj 

image

File: example/gen-config-yaml.sh (hunk 1 above)
Comment (gxthrj) [reply example/gen-config-yaml.sh:3]: yes, in docker-compose.yaml special the ip for etcd server

**Comment by gxthrj** [comment]
> @gxthrj
> 
> image

the comment has been changed

**Pull request merged**